src/createServer.js

```diff
@@ -1,3 +1,39 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { errorValidation } = require('./errorValidation');
+const { convertToCase } = require('./convertToCase/convertToCase');
+
+const createServer = () => {
+  const server = http.createServer((req, res) => {
+    const splitUrl = req.url.split('?');
+    const path = splitUrl[0];
+    const textToConvert = path.slice(1);
+    const queryStr = splitUrl[1];
+    const params = new URLSearchParams(queryStr);
+    const toCase = params.get('toCase');
```

Comment:
Let's decompose your logic
Make a separate fucntion for it
```suggestion
    const [..,..] = ...();
```